src/sinks/amqp/config.rs

```diff
@@ -18,6 +18,10 @@ pub struct AmqpPropertiesConfig {
     /// Content-Encoding for the AMQP messages.
     #[configurable(derived)]
     pub(crate) content_encoding: Option<String>,
+
+    /// Expiration for AMQP messages (in milliseconds)
+    #[configurable(derived)]
```

Comment:
```suggestion
```

This is not actually needed. The option is used by our config macro to pull the docs in from the type specified by the property. In this case we are providing the docs and `String` doesn't have any docs attached to it as it is a `stdlib` type.

The other properties above are also erroneously specifying this. Would you be able to remove the attribute from all the fields in this struct as well?